test/unit/rules/no-implicit-this-test.js

```diff
@@ -94,5 +95,24 @@ generateRuleTests({
         column: 10,
       },
     },
+    {
+      template: '<MyComponent @prop={{can.do}} />',
+      result: {
+        message: message('can.do'),
+        source: 'can.do',
+        line: 1,
+        column: 21,
+      },
+    },
+    {
+      template: '<MyComponent @prop={{can.do}} />',
```

Comment:
Note: this is still a failure (even when `can` is explicitly provided as an allowed helper) because it is definitely not a helper invocation.